neo/Network/P2P/TaskManager.cs

```diff
@@ -252,6 +260,11 @@ private void RequestTasks(TaskSession session)
                 }
                 session.RemoteNode.Tell(Message.Create("getblocks", GetBlocksPayload.Create(hash)));
             }
+            else if (Blockchain.Singleton.HeaderHeight >= session.LastBlockIndex
+                    && TimeProvider.Current.UtcNow.ToTimestamp() - PingCoolingOffPeriod >= Blockchain.Singleton.GetBlock(Blockchain.Singleton.CurrentHeaderHash)?.Timestamp)
```

Comment:
In fact, the previous check of all sessions was not so good...ahauahah

 I like this new logic, more clear!

Congratz.